prisma/migrations/20240103181300_add_dissolve_reasons_array/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "match" ADD COLUMN     "dissolveReasons" "dissolve_reason"[] DEFAULT ARRAY[]::"dissolve_reason"[];
```

Comment:
Would be easier to just UPDATE here and then directly drop the old column, but if you prefer to do it in a separate step thats also fine